doc/nrf/libraries/modem/at_shell.rst

```diff
@@ -0,0 +1,60 @@
+.. _lib_at_shell:
+
+AT shell
+########
+
+.. contents::
+   :local:
+   :depth: 2
+
+You can use this library to add an AT shell command to a shell.
+This enables you to send AT commands to the modem and receive responses.
+
+Configuration
+*************
+
+The library expects that you have enabled a shell using :kconfig:`CONFIG_SHELL`.
+To enable and configure the AT shell, use the following Kconfig options:
+
+* :kconfig:`CONFIG_AT_SHELL` - Enables the AT Host library
+* :kconfig:`CONFIG_AT_SHELL_CMD_RESPONSE_MAX_LEN` - Maximum AT command response length
+
+To send AT read commands (ending with ``?``), disable the shell wildcards using the following Kconfig option:
+
+* :kconfig:`CONFIG_SHELL_WILDCARD`
+
+Dependencies
```

Comment:
I would rather have the Usage section before the Dependencies.

Reply:
Fixed